src/wmnds/patterns/banner/_banner.njk

```diff
@@ -0,0 +1,12 @@
+{% macro wmndsBanner(params) %}
+
+<div class="wmnds-grid wmnds-banner-container">
+<div class="wmnds-col-1-8">
```

Comment:
This col is too small on mobile and crushes the beta/alpha tag. 
I'd change this to wmnds-col-auto